File: 2D-World-Simulator-Java/OPP-Java/src/op/JavaProject/Organisms/Animals/Sheep.java
package op.JavaProject.Organisms.Animals;

import op.JavaProject.Functionality.BoardPoint;
import op.JavaProject.Functionality.Species;
import op.JavaProject.Game.Animal;
import op.JavaProject.Game.World;

import java.awt.*;

public class Sheep extends Animal {

    public Sheep(World organismWorld, BoardPoint point) {
        super(organismWorld, point);
        this.strength = 4;
        this.initiative = 4;
        this.organismSign = "S";
        this.organismSpecie = Species.SHEEP;
        this.organismColor = new Color(227, 227, 227);
    }
}

File: 2D-World-Simulator-Python/OPP-Python/Organisms/Animals/Sheep.py
from Game.Animal import Animal
from Game.OrganismStatics import OrganismSpecie


class Sheep(Animal):
    def __init__(self, organism_world, point):
        super().__init__(organism_world, point, 4, 4, "S", OrganismSpecie.SHEEP, "#e3e3e3")
